A3AE/addons/templates/Templates/#Examples/RebelExample.sqf

```diff
@@ -2,73 +2,104 @@
-["staticMortar", ""] call _fnc_saveToTemplate;                 //this line determines static mortars -- Example: ["staticMortar", ["B_Mortar_01_F"]] -- String, can only use one
-["staticMortarMagHE", ""] call _fnc_saveToTemplate;
-["staticMortarMagSmoke", ""] call _fnc_saveToTemplate;
+["staticMGs", []] call _fnc_saveToTemplate;    //this line determines static MGs -- Array, can have multiple entries
+["staticAT", []] call _fnc_saveToTemplate;    //this line determinesstatic ATs -- Array, can have multiple entries
+private _staticAA = [];    //this line determines static AAs -- Array, can have multiple entries
+["staticMortars", []] call _fnc_saveToTemplate;    //this line determines static mortars -- Array, can have multiple entries
+["staticMortarMagHE", ""] call _fnc_saveToTemplate;    //this line determines HE shells for static mortars - String, single entry
+["staticMortarMagSmoke", ""] call _fnc_saveToTemplate;    //this line determines smoke shells static mortars - String, single entry
+["staticMortarMagFlare", ""] call _fnc_saveToTemplate;    //this line determines flare shells static mortars - String, single entry
 
-["mineAT", ""] call _fnc_saveToTemplate;                 //this line determines AT mines needed for spawning in minefields -- Example: ["minefieldAT", ["ATMine_Range_Mag"]] -- String, can only use one
-["mineAPERS", ""] call _fnc_saveToTemplate;             //this line determines APERS mines needed for spawning in minefields -- Example: ["minefieldAPERS", ["APERSMine_Range_Mag"]] -- String, can only use one
+["mineAT", ""] call _fnc_saveToTemplate;    //this line determines AT mines needed for spawning in minefields - String, single entry
+["mineAPERS", ""] call _fnc_saveToTemplate;    //this line determines APERS mines needed for spawning in minefields - String, single entry
 
-["breachingExplosivesAPC", []] call _fnc_saveToTemplate;            //this line determines explosives needed for breaching APCs -- Example: ["breachingExplosivesAPC", [["DemoCharge_Remote_Mag", 1]]] -- Array, can use Multiple
-["breachingExplosivesTank", []] call _fnc_saveToTemplate;           //this line determines explosives needed for breaching Tanks -- Example: [["SatchelCharge_Remote_Mag", 1], ["DemoCharge_Remote_Mag", 2]]] -- Array, can use Multiple
+["breachingExplosivesAPC", []] call _fnc_saveToTemplate;    //this line determines explosives needed for breaching APCs -- Array, can have multiple entries
+["breachingExplosivesTank", []] call _fnc_saveToTemplate;    //this line determines explosives needed for breaching Tanks -- Array, can have multiple entries
+
+if (allowDLCExpansion) then {
+    _vehiclesCivCar append [];
+};
+["vehiclesCivCar", _vehiclesCivCar] call _fnc_saveToTemplate;
+
+if (allowDLCWS) then {
+  _vehicleAA append [];
+  _staticAA insert [0, []];
+};
+["vehiclesAA", _vehicleAA] call _fnc_saveToTemplate;
+["staticAA", _staticAA] call _fnc_saveToTemplate;
+
+#include ".sqf"    //include vehicle attribute file
```

Comment:
Also skip these unless we're adding a vehicle attributes example, which we probably should.